### ISSUE TITLE
get_clips and get_total_reads produce JSON non-seriarizable 

### ISSUE THREAD

**Issue by chanshing** [activityMonitoring/biobankAccelerometerAnalysis#90]
@smallsr Scott Small encountered errors while processing his files, which I traced back to `get_clips` and `get_total_reads` https://github.com/activityMonitoring/biobankAccelerometerAnalysis/commit/42871e33f5897aeadba5fcfe8bb79d25d5e5e3ef

The problem is that `np.sum(e['clipsBeforeCalibr'])` etc. sometimes gives us a `numpy.float64` type (when there are NaNs in the column, e.g. missing values, interrupts), and sometimes a `numpy.int64` type (when no NaNs). Turns out the `json` package can't serialize `numpy.int64` types for some reason, resulting in `TypeError: Object of type int64 is not JSON serializable` error.

A quick fix is as follows

```python
def get_clips(e, epochPeriod, summary):
    summary['clipsBeforeCalibration'] = np.sum(e['clipsBeforeCalibr']).item()
    summary['clipsAfterCalibration'] = np.sum(e['clipsAfterCalibr']).item()

def get_total_reads(e, epochPeriod, summary):
    summary['totalReads'] = np.sum(e['rawSamples']).item()
```
Basically, adding a `.item()` call to convert from `numpy.*` type to native type (e.g. `numpy.int64` to native `int`, and `numpy.float64` to native `float`). Alternatively one can use `accUtils.formatNum` as it is done in `summariseEpoch.writeMovementSummaries()`.

I can submit a PR later unless someone is free to do it right away 👯
